tests/ui/test_ui_tools.py

```diff
@@ -2137,6 +2137,7 @@ def test_keypress_EDIT_MESSAGE(self, mocker, message_fixture,
     ])
     def test_transform_content(self, mocker, raw_html, expected_content,
                                messages_successful_response):
+        expected_content = expected_content.replace('{}', QUOTED_TEXT_PADDING)
```

Comment:
It is smaller than using a word, and `{}` automatically suggests that there is something to be substituted here, since `.format()` using the same notation. So it will make the parameters easier to understand.